templates/pages/login.html.twig

```diff
@@ -127,4 +127,4 @@
 $('#login_name').focus();
 });
 </script>
-{% endblock %}
+{% endblock %}
```

Comment:
Please avoid changes on unrelated files

Reply:
This change doesn't make sense, it tells me I've added something, but it's already there.

Reply:
Fixed in 1190d286e236a83d46c8caa1cf00216794980e20